test/CMakeLists.txt

```diff
@@ -182,49 +192,47 @@ if(NRN_ENABLE_PYTHON AND PYTEST_FOUND)
       GROUP rxdmod_tests
       NAME rxd_tests
       PRELOAD_SANITIZER
-      COMMAND COVERAGE_FILE=.coverage.rxd_tests ${change_test_tolerance} ${PYTHON_EXECUTABLE}
-              ${pytest} ./test/rxd)
+      ENVIRONMENT COVERAGE_FILE=.coverage.rxd_tests ${change_test_tolerance}
+      COMMAND ${PYTHON_EXECUTABLE} ${pytest_nofork} ./test/rxd)
     if(NRN_ENABLE_MPI)
       nrn_find_python_module(mpi4py)
       if(mpi4py_FOUND)
         nrn_add_test(
           GROUP rxdmod_tests
           NAME rxd_mpi_tests
           PRELOAD_SANITIZER
-          COMMAND
-            COVERAGE_FILE=.coverage.rxd_mpi_tests ${change_test_tolerance} ${MPIEXEC_NAME}
-            ${MPIEXEC_NUMPROC_FLAG} 1 ${MPIEXEC_PREFLAGS} ${PYTHON_EXECUTABLE} ${MPIEXEC_POSTFLAGS}
-            ${pytest} ./test/rxd --mpi)
+          ENVIRONMENT COVERAGE_FILE=.coverage.rxd_mpi_tests ${change_test_tolerance}
+          COMMAND ${MPIEXEC_NAME} ${MPIEXEC_NUMPROC_FLAG} 1 ${MPIEXEC_PREFLAGS}
+                  ${PYTHON_EXECUTABLE} ${MPIEXEC_POSTFLAGS} ${pytest_nofork} ./test/rxd --mpi)
       endif()
     endif()
   endif()
   if(NRN_ENABLE_MPI)
     add_test(
       NAME parallel_tests
       COMMAND
-        ${CMAKE_COMMAND} -E env ${MPIEXEC_NAME} ${MPIEXEC_NUMPROC_FLAG} 6 ${MPIEXEC_OVERSUBSCRIBE}
-        ${MPIEXEC_PREFLAGS} ${CMAKE_BINARY_DIR}/bin/nrniv ${MPIEXEC_POSTFLAGS} -mpi -python
+        ${MPIEXEC_NAME} ${MPIEXEC_NUMPROC_FLAG} 6 ${MPIEXEC_OVERSUBSCRIBE} ${MPIEXEC_PREFLAGS}
```

Comment:
I think the code was the way it was because of https://github.com/neuronsimulator/nrn/blob/4a5867f45e6e94f58be9df2826e73ab0972face1/CMakeLists.txt#L312-L315 🤔